pkg/test/verify/cluster.go

```diff
@@ -210,16 +191,8 @@ func DeleteToolchainCluster(t *testing.T, functionToVerify FunctionToVerify) {
 	assert.Nil(t, cachedToolchainCluster)
 }
 
-func Labels(clType cluster.Type, ns, ownerClusterName string) map[string]string {
+func Labels(ns, ownerClusterName string) map[string]string {
 	labels := map[string]string{}
-	if clType != "" {
-		labels["type"] = string(clType)
-		// Set cluster role tenant label only for member type clusters.
-		if clType == cluster.Member {
-			// We use only the label key, the value can remain empty.
-			labels[cluster.RoleLabel(cluster.Tenant)] = ""
-		}
```

Comment:
don't we want to keep setting this role for toolchaincluster representing members ?

Reply:
nope, as mentioned in the other comments, this is done in add-cluster.sh script.